src/go/k8s/pkg/resources/configmap.go

```diff
@@ -88,12 +123,52 @@ func (r *ConfigMapResource) Ensure(ctx context.Context) error {
 			return err
 		}
 
+		if r.pandaCluster.Spec.ExternalConnectivity {
+			obj.(*corev1.ConfigMap).Data[nodeIPsList] = IPsList.String()
+		}
+
 		return r.Create(ctx, obj)
 	}
 
+	if r.pandaCluster.Spec.ExternalConnectivity && !reflect.DeepEqual(cfgm.Data[nodeIPsList], IPsList.String()) {
+		cfgm.Data[nodeIPsList] = IPsList.String()
+		if err := r.Update(ctx, &cfgm); err != nil {
+			return fmt.Errorf("failed to update ConfigMap: %w", err)
+		}
+	}
+
 	return nil
 }
 
+func getNodePort(svc *corev1.Service) int32 {
```

Comment:
why is this duplicated from https://github.com/vectorizedio/redpanda/pull/694/files#diff-add8bf816971d3e4c76f469321f420a65afa670f817e61c516bcf769afe09e07R40

Reply:
Thanks for the finding. I will remove duplicated function.